src/actions/totalOrgSummary.js

```diff
@@ -31,7 +31,7 @@ export const fetchTotalOrgSummaryReportError = error => ({
 export const getTotalOrgSummary = (startDate, endDate) => {
   const url = ENDPOINTS.TOTAL_ORG_SUMMARY(startDate, endDate);
   return async dispatch => {
-    dispatch(fetchTotalOrgSummaryReportBegin());
+    await dispatch(fetchTotalOrgSummaryReportBegin());
```

Comment:
The return value of promise is not saved in any constant.  what is the use of using await operator?

Reply:
Actually, that code is not involved in this feature enhancement. It was a typo that will be fixed. Thanks.